Pulse/dist/metadata.json

```diff
@@ -10,7 +10,7 @@
     "service": "Pulse",
     "version": "1.0.0",
     "logo": "https://i.imgur.com/mqYHGOC.png",
-    "thumbnail": "https://cdn.discordapp.com/attachments/630142339872194591/680154520919081024/unknown.png",
+    "thumbnail": "https://i.imgur.com/eeZ8Wlm.png",
```

Comment:
uh
> The media you include (icon/thumbnail) must be related to the website and should be understandable in terms of size and quality.

Reply:
ofc I am the stupid idiot that used the wrong link HAHA